public/locales/es/claim-status.json

```diff
@@ -84,7 +84,8 @@
     }
   },
   "conditional-next-steps": {
-    "certify-no-pending": "<0>Siga certificando</0> mientras necesite beneficios.",
-    "certify-pending": "<0>Siga certificando</0> mientras necesite beneficios, para que sus pagos no se retrasen más."
+    "continue-certifying": {
+      "text": "Continúe <0>certificando los beneficios</0> mientras determinamos su elegibilidad."
```

Comment:
No. I'm using `public/locales/en/claim-status.json` as the key document. Because the translation string keys would be the same in _every_ locale doc, duplicating them in each locale doc could introduce additional room for errors/typos. I'll clarify that in the Foundation doc.